src/main/java/by/itech/lab/supplier/repository/WarehouseRepository.java

```diff
@@ -0,0 +1,10 @@
+package by.itech.lab.supplier.repository;
+
+import by.itech.lab.supplier.domain.Warehouse;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface WarehouseRepository extends JpaRepository<Warehouse, Long> {
```

Comment:
Should it be marked as `@Repository?`

Reply:
+

Reply:
I cant see you "done" until you have not uploaded it here